docker-compose.yml

```diff
@@ -50,15 +50,54 @@ services:
       - prover
       - proposer
 
+  l2_execution_engine_taiko_reth:
+    image: ghcr.io/tatsujinlabs/taiko-reth:nightly
+    restart: unless-stopped
+    pull_policy: always
+    platform: linux/amd64
+    stop_grace_period: 3m
+    volumes:
+      - l2_execution_engine_data:/data/taiko-reth
+    ports:
+      - "${PORT_L2_EXECUTION_ENGINE_METRICS}:6060"
+      - "${PORT_L2_EXECUTION_ENGINE_HTTP}:8545"
+      - "${PORT_L2_EXECUTION_ENGINE_WS}:8546"
+      - "${PORT_L2_EXECUTION_ENGINE_P2P}:${PORT_L2_EXECUTION_ENGINE_P2P}"
+      - "${PORT_L2_EXECUTION_ENGINE_P2P}:${PORT_L2_EXECUTION_ENGINE_P2P}/udp"
+    command: |
+      ./taiko-reth node
+        --http
+        --http.addr 0.0.0.0
+        --ws
+        --ws.addr 0.0.0.0
+        --authrpc.addr 0.0.0.0
+        --chain mainnet
```

Comment:
are we really ready to use it for mainnet? I haventnfollowed the taiko-geth progress thus far.